src/component/parser/xml/bpmn-json-model/baseElement/flowNode/event.ts

```diff
@@ -60,9 +60,6 @@ export interface TCatchEvent extends TEvent {
 
   eventDefinitionRef?: string | string[];
   parallelMultiple?: boolean; // default="false"
-
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
:question: I imagine this is not used anymore, isn't it?

Reply:
I moved it in the parent class TEvent.

Reply:
ok thx